lib/generators/rails_template18f/terraform/templates/terraform/production/main.tf.tt

```diff
@@ -5,32 +5,48 @@ locals {
   app_name      = "<%= app_name %>"
 }
 
+module "app_space" {
+  source = "github.com/gsa-tts/terraform-cloudgov//cg_space?ref=v1.1.0"
+
+  cf_org_name   = local.cf_org_name
+  cf_space_name = local.cf_space_name
+  deployers     = [var.cf_user]
+  # developers should include any users that will potentially need to run `cf ssh` on the app
+  developers = []
+}
+
 module "database" {
-  source = "github.com/gsa-tts/terraform-cloudgov//database?ref=v1.0.0"
+  source = "github.com/gsa-tts/terraform-cloudgov//database?ref=v1.1.0"
 
   cf_org_name   = local.cf_org_name
   cf_space_name = local.cf_space_name
   name          = "${local.app_name}-rds-${local.env}"
   rds_plan_name = "TKTK-production-rds-plan"
+  # depends_on line is needed only for initial creation and destruction. It should be commented out for updates to prevent unwanted cascading effects
+  depends_on = [module.app_space]
```

Comment:
Suggestion (non-blocking): Based on the comment, this might be useful to ignore any updates and just create/destroy the resource. Gory details here: https://spacelift.io/blog/terraform-ignore-changes.

Unfortunately this can't be done in a module block at this time. https://github.com/hashicorp/terraform/issues/27360 The `database` module would need to updated to use a count/other technique to add a block like:

```
  # Ignoring updates to prevent unwanted cascading effects
  lifecycle {
    ignore_changes = all
  }
```